<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo to 3D</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenCGA"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionGenerateTrainingImages"/>
    <addaction name="actionParameterEstimation"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionCameraStreetView"/>
    <addaction name="actionCameraAerialView"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionParameterEstimation">
   <property name="text">
    <string>Parameter Estimation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpenCGA">
   <property name="text">
    <string>Open CGA</string>
   </property>
  </action>
  <action name="actionOpenContour">
   <property name="text">
    <string>Open Contour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveContour">
   <property name="text">
    <string>Save Contour</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionExtractCameraParameter">
   <property name="text">
    <string>Extract Camera Parameter</string>
   </property>
  </action>
  <action name="actionParameterEstimationWithCameraCalibration">
   <property name="text">
    <string>Parameter Estimation with Camera Calibration</string>
   </property>
  </action>
  <action name="actionGenerateTrainingImages">
   <property name="text">
    <string>Generate Training Images</string>
   </property>
  </action>
  <action name="actionGenerateTrainingImages2">
   <property name="text">
    <string>Generate Training Images2</string>
   </property>
  </action>
  <action name="actionCameraStreetView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Street View</string>
   </property>
  </action>
  <action name="actionCameraAerialView">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Aerial View</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
